### ISSUE TITLE
[Issue] Wrong or broken verification link

### ISSUE THREAD

**Issue by rabrunos** [Sinhika/SimpleOres2#58]
From the looks of it, the correct link is [1.18](https://raw.githubusercontent.com/Sinhika/SimpleOres2/1.18/update.json) and not [1.18.2](https://raw.githubusercontent.com/Sinhika/SimpleOres2/1.18.2/update.json)

```ruby
[15:23:47] [Forge Version Check/INFO] [ne.mi.fm.VersionChecker/]: [simpleores] Starting version check at https://raw.githubusercontent.com/Sinhika/SimpleOres2/1.18.2/update.json
[15:23:47] [Forge Version Check/WARN] [ne.mi.fm.VersionChecker/]: Failed to process update information
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was NUMBER at line 1 column 4 path $
	at com.google.gson.Gson.fromJson(Gson.java:975) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.Gson.fromJson(Gson.java:928) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.Gson.fromJson(Gson.java:877) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.Gson.fromJson(Gson.java:848) ~[gson-2.8.9.jar%2319!/:?] {}
	at net.minecraftforge.fml.VersionChecker$1.process(VersionChecker.java:189) ~[fmlcore-1.18.2-40.0.12.jar%2377!/:?] {}
	at java.lang.Iterable.forEach(Iterable.java:75) [?:?] {re:mixin}
	at net.minecraftforge.fml.VersionChecker$1.run(VersionChecker.java:128) [fmlcore-1.18.2-40.0.12.jar%2377!/:?] {}
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was NUMBER at line 1 column 4 path $
	at com.google.gson.stream.JsonReader.beginObject(JsonReader.java:384) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:183) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.internal.bind.MapTypeAdapterFactory$Adapter.read(MapTypeAdapterFactory.java:145) ~[gson-2.8.9.jar%2319!/:?] {}
	at com.google.gson.Gson.fromJson(Gson.java:963) ~[gson-2.8.9.jar%2319!/:?] {}
	... 6 more
```

**Comment by Sinhika** [comment]
Fixed with release 1.18.2-4.1.0.2 https://www.curseforge.com/minecraft/mc-mods/simpleores/files/3695702.

**Comment by rabrunos** [comment]
Great job!
